lib/Doctrine/Import/Mssql.php

```diff
@@ -219,10 +219,10 @@ public function listTableViews($table)
         }
         $table = $this->conn->quote($table, 'text');
         $query = 'EXEC sp_statistics @table_name = ' . $this->conn->quoteIdentifier($table, true);
-        $indexes = $this->conn->fetchColumn($query, $keyName);
+        $indexes = $this->conn->fetchColumn($query, array($keyName));
 
         $query = 'EXEC sp_pkeys @table_name = ' . $this->conn->quoteIdentifier($table, true);
-        $pkAll = $this->conn->fetchColumn($query, $pkName);
+        $pkAll = $this->conn->fetchColumn($query, array($pkName));
```

Comment:
This should be an array, I followed it through and ends up being iterated here: https://github.com/diablomedia/doctrine1/blob/master/lib/Doctrine/Connection/Statement.php#L243